library-generation-unit-tests failed in release PR

library-generation-unit-tests failed in release PR due to the following 
```
FAIL: test_monorepo_postprocessing_valid_repository_success (library_generation.tests.utils.monorepo_postprocessor_unit_tests.MonorepoPostprocessorTest.test_monorepo_postprocessing_valid_repository_success)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/google-cloud-java/google-cloud-java/sdk-platform-java/hermetic_build/library_generation/tests/utils/monorepo_postprocessor_unit_tests.py", line 42, in test_monorepo_postprocessing_valid_repository_success
    file_comparator.compare_files(
  File "/home/runner/work/google-cloud-java/google-cloud-java/sdk-platform-java/hermetic_build/library_generation/tests/test_utils.py", line 29, in compare_files
    self.assertEqual(
AssertionError: Lists differ: [] != ['--- \n', '+++ \n', '@@ -24,19 +24,19 @@\[1232 chars]>\n']

Second list contains 25 additional elements.
First extra element 0:
'--- \n'

Diff is 1395 characters long. Set self.maxDiff to None to see it. : Unexpected file contents:
--- 
+++ 
@@ -24,19 +24,19 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-dns</artifactId>
-        <version>2.33.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-dns:current} -->
+        <version>2.89.0</version><!-- {x-version-update:google-cloud-dns:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-service-control-bom</artifactId>
-        <version>1.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-service-control:current} -->
+        <version>1.91.0</version><!-- {x-version-update:google-cloud-service-control:current} -->
         <type>pom</type>
         <scope>import</scope>
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-tasks-bom</artifactId>
-        <version>2.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-tasks:current} -->
+        <version>2.91.0</version><!-- {x-version-update:google-cloud-tasks:current} -->
         <type>pom</type>
         <scope>import</scope>
       </dependency>
```

See [log](https://github.com/googleapis/google-cloud-java/actions/runs/24373536633/job/71182415094) for details.

We likely need to exclude this file from updating by release-please.

--- a/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-dns/pom.xml
+++ b/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-dns/pom.xml
@@ -4,6 +4,6 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-dns</artifactId>
   <packaging>jar</packaging>
-  <version>2.33.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-dns:current} -->
+  <version>2.33.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-dns-golden:current} -->
   <name>Google Cloud DNS Parent</name>
 </project>

--- a/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-service-control/google-cloud-service-control-bom/pom.xml
+++ b/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-service-control/google-cloud-service-control-bom/pom.xml
@@ -3,6 +3,6 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-service-control-bom</artifactId>
-  <version>1.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-service-control:current} -->
+  <version>1.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-service-control-golden:current} -->
   <packaging>pom</packaging>
 </project>

--- a/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-tasks/google-cloud-tasks-bom/pom.xml
+++ b/sdk-platform-java/hermetic_build/library_generation/tests/resources/test_monorepo_postprocessing/java-tasks/google-cloud-tasks-bom/pom.xml
@@ -3,6 +3,6 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-tasks-bom</artifactId>
-  <version>2.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-tasks:current} -->
+  <version>2.35.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-tasks-golden:current} -->
   <packaging>pom</packaging>
 </project>

